tests/conftest.py

```diff
@@ -487,7 +488,8 @@ def assert_app_id(application_pid: int, runtime: str, in_container: bool) -> Gen
     desired_name_and_getter = {
         "java": (get_java_app_id, "java: Fibonacci.jar"),
         "python": (get_python_app_id, "python: lister.py (/app/lister.py)"),
-        "node": (get_node_app_id, "nodejs: fibonacci.js (/app/fibonacci.js"),
```

Comment:
How did we merge it this way o.O I mean how did the tests pass

Reply:
So this was never tested :(

Reply:
Ohh.